docs/awesome-babel/babel-plugin-add-module-exports.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: docs
```

Comment:
I'm not sure that Jekyll will allow to include remote content (specialy using GitHub pages).